src/prachi/Assignment_15/AccountSwitch.java

```diff
@@ -0,0 +1,174 @@
+package prachi.Assignment_15;
+/*Assignment - 15 : 9th April'2021
+
+A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+a) Person can debit or credit amount from respective bank.
+Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+c) A Method to show total credit, debit operations
+output : Total credit operation - 7 times
+Total debit operation - 5 times
+
+d) A Method to show individual bank credit operations.
+output : ICICI bank Credit Operation - 2
+HDFC bank Credit Operation - 2
+SBI bank Credit Operation - 1
+BOB bank Credit Operation - 4
+
+e) A Method to show individual bank debit operations.
+output : ICICI bank Debit Operation - 2
+HDFC bank Debit Operation - 2
+SBI*/
+
+public class AccountSwitch {
+
+	static int iciciBal = 10000;
+	static int hdfcBal = 15000;
+	static int bobBal = 20000;
+	static int sbiBal = 25000;
+	
+	static int totalDebitCount;
+	static int totalCreditCount;
+
+	int iciciCredCnt=0, hdfcCredCnt=0, bobCredCnt=0, sbiCredCnt=0;
+	int iciciDebCnt=0, hdfcDebCnt=0, bobDebCnt=0, sbiDebCnt=0;
+
+	void operation(String bank, String op, int amt) {
+		switch (op) {
+		case "debit":
+			switch (bank) {
+			case "icici":
+				if (iciciBal>=amt) {
+					iciciBal=iciciBal-amt;
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.